arelle/plugin/csvTestcaseLoader.py

```diff
@@ -0,0 +1,114 @@
+"""
+See COPYRIGHT.md for copyright information.
+
+## Overview
+
+This plugin determines whether the subject file is a CSV testcase file,
+and if so returns a ModelDocument with ModelTestcaseObjects as if loaded from xml
+
+Note: please add expected testcase error xmlns elements to testcaseElementXml below
+
+"""
+
+import csv, io, os
```

Comment:
Plugins for running tests have been moved to [tests/plugin](https://github.com/Arelle/Arelle/tree/master/tests/plugin). This should go in there, too.